Formula/m/morpheus.rb

```diff
@@ -0,0 +1,111 @@
+class Morpheus < Formula
+  desc "Modelling environment for multi-cellular systems biology"
+  homepage "https://morpheus.gitlab.io/"
+  url "https://gitlab.com/morpheus.lab/morpheus/-/archive/v2.3.6/morpheus-v2.3.6.tar.gz"
+  sha256 "abb277f3898467b51994e39ed0f4b38fc9aec79b98b126c6d17d971716612d82"
+  license "BSD-3-Clause"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+(?:_?\d+)?)$/i)
+  end
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "ninja" => :build
+  depends_on "ffmpeg"
+  depends_on "graphviz"
+  depends_on "libomp"
+  depends_on "libtiff"
+  depends_on "qt@5"
```

Comment:
any issue of tracking qt6 support?

Reply:
Morpheus doesn't support Qt6 yet, so there is unfortunately no other option than Qt5 at the moment, but we're working on that and we'll be adding Qt6 support in an upcoming version.